src/styles/main.scss

```
@@ -1,3 +1,75 @@
+$weekdays: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6
+);
+$month-lengths: 28, 29, 30, 31;
+$calendar-day-gap: 1px;
+$calendar-day-width: 100px - $calendar-day-gap;
```

Move it to separate variables file
```suggestion
$weekdays: (
  'mon': 0,
  'tue': 1,
  'wed': 2,
  'thu': 3,
  'fri': 4,
  'sat': 5,
  'sun': 6
);
$month-lengths: 28, 29, 30, 31;
$calendar-day-gap: 1px;
$calendar-day-width: 100px - $calendar-day-gap;
```